http/mdm/mdm.go

```diff
@@ -37,12 +37,19 @@ func CheckinHandler(svc service.Checkin, logger log.Logger) http.HandlerFunc {
 		}
 		respBytes, err := service.CheckinRequest(svc, mdmReqFromHTTPReq(r), bodyBytes)
 		if err != nil {
-			logger.Info("msg", "check-in request", "err", err)
+			logs := []interface{}{"msg", "check-in request", "err", err}
```

Comment:
if we log the error here, it'll contain the longer error which includes the body. since the point (I think) is to log the body separately and then we'd want to avoid the duplicate logging. we'd want to unwrap the error before logging it (since we're logging its properties, anyway). not sure if i'm explaining it right.